lib/modules/nightMode.js

```diff
@@ -117,6 +119,14 @@ addModule('nightMode', function(module, moduleID) {
 			);
 		}
 	},
+	afterLoad: function () {
+		// after loading, create event handler for when neverEndingEeddit finishes a page.
+		window.addEventListener('neverEndingLoad', modules['nightMode'].afterNeverEndingLoad);
```

Comment:
It would be tidier to listen for RES's siteTable.. syntax is something like `RESUtils.watch.siteTable.add(callback)` -- I can grab it tonight if you don't find it first.